modules/haloPlaylist.js

```diff
@@ -7,14 +7,14 @@ async function getPlaylist(request, response) {
     const appid = process.env.HALO_API_KEY;
 
     try {
-        const playlistResults = await axios.get('https://www.haloapi.com/stats/h5/player-leaderboards/csr/46f2fe0c-8478-4593-82a3-bfb01c1cd63f/d34730b9-bb48-4569-bcdf-afba9c358019'), {
-            header:  {'Ocp-Apim-Subscription-Key':  appid},
-        };
+        const playlistResults = await axios.get('https://www.haloapi.com/stats/h5/player-leaderboards/csr/46f2fe0c-8478-4593-82a3-bfb01c1cd63f/d34730b9-bb48-4569-bcdf-afba9c358019', {
+            headers:  {'Ocp-Apim-Subscription-Key': appid},
+    });
 
 
-    console.log('Here is the playlist info.........................................................!!', playlistResults.data.playlists);
+    console.log('Here is the playlist info.........................................................!!', playlistResults.data.results);
 
-    let playlistData = playlistResults.data.playlists.map(playlist => new playlist(playlist));
+    let playlistData = playlistResults.data.results.map(player => new Playlist(playlist));

```

Comment:
You said `data` is shaped like this:

```json
[
    {
        "playlists": [
            {
                "name": "Team Arena",
```

I don't see `results` in here. So the question is: what is the outer array and how do we get to the `playlists` we're looking for?